<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-4CJM1GA" failures="2" tests="4" name="PetTests" time="4.844" errors="0" timestamp="2025-04-17T17:25:20 IST">
  <testcase classname="api.test.UserTests" name="testPostuser" time="2.107"/>
  <testcase classname="api.test.UserTests" name="testGetUserbyName" time="0.555">
    <failure type="java.lang.AssertionError" message="expected [200] but found [404]">
      <![CDATA[java.lang.AssertionError: expected [200] but found [404]
at api.test.UserTests.testGetUserbyName(UserTests.java:56)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
... Removed 36 stack frames]]>
    </failure>
  </testcase> <!-- testGetUserbyName -->
  <testcase classname="api.test.UserTests" name="testUpdateUserByName" time="1.02"/>
  <testcase classname="api.test.UserTests" name="testDeleteUserbyName" time="0.489">
    <failure type="java.lang.AssertionError" message="expected [200] but found [404]">
      <![CDATA[java.lang.AssertionError: expected [200] but found [404]
at api.test.UserTests.testDeleteUserbyName(UserTests.java:87)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
... Removed 36 stack frames]]>
    </failure>
  </testcase> <!-- testDeleteUserbyName -->
</testsuite> <!-- PetTests -->
